pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -242,6 +242,25 @@ func (m *manager) reconcileState() (success []reconciledContainer, failure []rec
 				continue
 			}
 
+			// Check whether container is present in state, there may be 3 reasons why it's not present:
+			// - container is in Be/Bu pod
+			// - kubelet just restarted - state has not been restored properly and is empty
+			// - container has been removed from state by RemoveContainer call (DeletionTimestamp is set)
```

Comment:
Does this case apply? In case deletion timestamp is set, does the pod appear in the active pod list that we are currently iterating over?

Reply:
yes, the pod is visible